reinforcement_learning/rl_mujoco_ray/rl_hopper_ray_gymEnv.ipynb

```diff
@@ -0,0 +1,534 @@
+{
```

Comment:
<p>Add context on Mujoco and Hopper - what do these libraries do, what domain are they in, who might this be relevant to, etc.</p><p> </p><p>Style nit - use all-caps "CPU" / "GPU" in markdown rather than "cpu"/"gpu"</p><p> </p><p>Grammar nit - <code>Tensorflow</code> -&gt; <code>TensorFlow</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Thanks, added.